src/main/resources/templates/fragments.html

```diff
@@ -27,15 +27,17 @@ <h1 class="navbar-brand" th:text="${title}">LRT</h1>
 			<li class="nav-item dropdown">
 				<a class="nav-link dropdown-toggle" id="navbarDropdownTwo" role="button" data-toggle="dropdown">Categories</a>
 				<div class="dropdown-menu">
-					<a class="dropdown-item" th:href="@{/categories}">View All</a>
+					<a class="dropdown-item" th:href="@{/categories}">View All / Edit / Copy</a>
```

Comment:
This is a minor point, and doesn't necessarily necessitate a change, but do we need to add the /edit/copy to the view all here? When we open up the list of recipes we can see the functionality present in each list so I don't know if we need to state it so explicitly in the drop-down.